### ISSUE TITLE
How to multiply 2 matrices of the same size element-wise using Keras backend?

### ISSUE THREAD

**Issue by daolamgit** [keras-team/keras#9042]
I am writing a custom loss function but couldn't find the function do perform element-wise multiplication. Any suggestion?
Thanks

**Comment by daolamgit** [comment]
Answer (myself): *

**Comment by maulberto3** [comment]
@daolamgit  Hi, did you find it?
